Gemfile

```diff
@@ -0,0 +1,13 @@
+source "https://rubygems.org"
+
+gem 'emeril'
```

Comment:
Since all this gem is only necessary to release the cookbook (and not to use it), should we put it (and all other `rake release` dependencies) in a group? For example:
```ruby
group :release do
  gem 'emeril'
  ...
end
```
This makes it clear that they're not needed to use the cookbook.